insolar/bus/bus.go

```diff
@@ -479,13 +479,14 @@ func (b *Bus) wrapMeta(
 		ID:         []byte(msg.UUID),
 	}
 
-	buf, err := meta.Marshal()
+	buf, err := payloadMeta.Marshal()
 	if err != nil {
 		return payload.Meta{}, nil, errors.Wrap(err, "wrapMeta. failed to wrap message")
 	}
 	msg.Payload = buf
+	msg.Metadata.Set(meta.Receiver, receiver.String())
```

Comment:
@megge-dream for me this one actually looks good, I wonder if we can switch to medata field?